libnmstate/ifaces/ifaces.py

```diff
@@ -325,6 +326,17 @@ def _validate_unknown_parent(self):
                     f"{iface.parent}"
                 )
 
+    def _remove_unknown_interfaces(self):
+        """
+        Remove unknown type interfaces that come from the current state.
+        """
+        for iface in self.current_ifaces.values():
+            if (
+                not iface.is_desired
```

Comment:
Not require for checking `not iface.is_desired`.

Reply:
Removed, thanks!